ansible/roles/network/tasks/vpn-client.yml

```diff
@@ -0,0 +1,27 @@
+---
+- when: liquid_vpn.client
+  block:
+
+  - name: OpenVPN client configuration file
+    copy:
+      content: "{{ liquid_vpn.client.config_file}}"
```

Comment:
I think this variable (`liquid_vpn.client.config_file`) should contain "content" in its name, because it's a string with the file content. Merge it as it is, and we'll refactor it after the milestone.